feature_layers/feature-collection-layer-from-portal/src/main/java/com/esri/samples/feature_collection_layer_from_portal/FeatureCollectionLayerFromPortalSample.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.feature_collection_layer_from_portal;
+
+import com.esri.arcgisruntime.data.FeatureCollection;
+import com.esri.arcgisruntime.layers.FeatureCollectionLayer;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.portal.Portal;
+import com.esri.arcgisruntime.portal.PortalItem;
+
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextField;
+import javafx.application.Application;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+
+
+public class FeatureCollectionLayerFromPortalSample extends Application {
+
+    private MapView mapView;
+    private ArcGISMap map;
+    private Portal portal;
+    private FeatureCollectionLayer featureCollectionLayer;
+    private FeatureCollection featureCollection;
+    private TextField input;
+
+    @Override
+    public void start(Stage stage) {
+        // set up the scene
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Feature Collection Layer From Portal");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // create amp and set it to be displayed in this view
+            map = new ArcGISMap(Basemap.createOceans());
+            // create a view for this ArcGISMap and set ArcGISMap to it
+            mapView = new MapView();
+            mapView.setMap(map);
```

Comment:
The comments here don't quite line up with the order of what the code is doing.
I've noticed other samples do this as well. I'll create an issue for us to tidy these up a little and make them more uniform.

For this sample, I'd suggest rearranging a bit:
```suggestion

            // create a new ArcGISMap with an oceans Basemap
            map = new ArcGISMap(Basemap.createOceans());
            
            // create a map view and set the ArcGISMap to it
            mapView = new MapView();
            mapView.setMap(map);
```